include/translated-object.php

```diff
@@ -428,13 +428,13 @@ public function where_clause( $lang ) {
 		 * $lang is a comma separated list of slugs ( or an array of slugs ).
 		 * This is generally the case is the query is coming from outside with a 'lang' parameter.
 		 */
-		$slugs     = is_array( $lang ) ? $lang : explode( ',', $lang );
-		$languages = array();
-		foreach ( $slugs as $slug ) {
-			$languages[] = absint( $this->model->get_language( $slug )->$tt_id );
+		$languages        = is_array( $lang ) ? $lang : explode( ',', $lang );
+		$languages_tt_ids = array();
+		foreach ( $languages as $language ) {
+			$languages_tt_ids[] = absint( $this->model->get_language( $language )->$tt_id );
 		}
```

Comment:
To make sure this won't come back to bite our asses some day:
```suggestion
		foreach ( $languages as $language ) {
			$language = $this->model->get_language( $language );

			if ( ! empty( $language ) ) {
				$languages_tt_ids[] = absint( $language->$tt_id );
			}
		}

		if ( empty( $languages_tt_ids ) ) {
			return '';
		}
```

Reply:
Ouch! Good catch!